### PR TITLE
Added more stats

### PR DESCRIPTION
**Pull request by auriv**
Added more stats

### CODE REVIEW

File: fftools/ffmpeg.c [hunk 1]
```diff
@@ -1743,19 +1743,33 @@ static void print_report(int is_last_report, int64_t timer_start, int64_t cur_ti
...
-    static int transcode_reset_frame_number = 0;
-    static int64_t transcode_reset_total = 0;
-    static int64_t transcode_us_last = 0;
-    static float prev_transcode_pf = 0.0;
-    static int64_t prev_transcode_max = 0;
-
-    if(transcode_us > 0) {
-        transcode_total += transcode_us;
-        transcode_max = transcode_us > transcode_max ? transcode_us : transcode_max;
-        transcode_us_last = transcode_us;
```
Comment (richardscreene) [inline fftools/ffmpeg.c:1758]: Can't most of these be uints?

File: fftools/ffmpeg.c [hunk 2]
```diff
@@ -1850,24 +1878,48 @@ static void print_report(int is_last_report, int64_t timer_start, int64_t cur_ti
...
+                cur_enc_pct = (enc_total - enc_reset_total) / total_us;
+                cur_dec_pct = (dec_total - dec_reset_total) / total_us;
+                prev_enc_pf = cur_enc_pf;
+                prev_dec_pf = cur_dec_pf;
+                prev_enc_max = cur_enc_max;
+                prev_dec_max = cur_dec_max;
+                prev_enc_pct = cur_enc_pct;
+                prev_dec_pct = cur_dec_pct;
+#if 0                
+                if(cur_fps<10) {
```
Comment (richardscreene) [inline fftools/ffmpeg.c:1906]: doesn't the format string expect a value?

File: fftools/ffmpeg.c [hunk 3]
```diff
@@ -5053,19 +5105,33 @@ static int transcode(void)
 /*Proximie*/
 #if 1
         {
-            static int last_total_frames = 0;
-            int64_t transcode_us = 0;
-            /* Calc stats only if we had at least 1 frame to encode */
```
Comment (richardscreene) [inline fftools/ffmpeg.c:5058]: now you seem to be mixing uints and ints ;-)

File: fftools/ffmpeg.c [hunk 4]
```diff
@@ -1821,9 +1835,23 @@ static void print_report(int is_last_report, int64_t timer_start, int64_t cur_ti
...
+            float cur_dec_pf;
+            int64_t cur_enc_max;
+            int64_t cur_dec_max;
+            float cur_enc_pct;
+            float cur_dec_pct;
+            int frames_decoded = 0;
+
+            for (int k = 0; k < nb_input_streams; k++) {
+                InputStream *ist = input_streams[k];
+                if (ist && ist->decoding_needed) {
```
Comment (richardscreene) [inline fftools/ffmpeg.c:1850]: if frames_decoded can be a uint64 then we can omit the cast

File: fftools/ffmpeg.c (hunk 1 above)
Comment (auriv) [reply fftools/ffmpeg.c:1758]: using same type as ffmpeg uses so dont need to be casting

**Review by richardscreene — APPROVED** [review]

File: fftools/ffmpeg.c (hunk 4 above)
Comment (auriv) [reply fftools/ffmpeg.c:1850]: agreed, and had it set same, but then reverted to keep with ffmpeg  "frame_number" int

File: fftools/ffmpeg.c (hunk 2 above)
Comment (auriv) [reply fftools/ffmpeg.c:1906]: yep.. fixed

**Pull request merged**